### PR TITLE
Open the project file for editing on double click or ENTER

### LINKED ISSUE #1224: Show project file itself in Solution Explorer
One of the big benefits of `project.json` was ease of editing. Unfortunately, that's now hidden behind a right click and a menu scan for the user. Here's what that looks like:
screen shot 2017-01-17 at 16 40 23

In the `project.json` world, this access for human editing was far easier:
screen shot 2017-01-17 at 16 48 34

Can we please add the `.csproj` back to the tree for easy editing? It could take many forms. The project.json was easy because the name was consistent, which would be a challenge here. Either it could be a special node always at the top, or some other access.

### PR DESCRIPTION
**Pull request by davidwengier**
Open project file for editing on double click. Fixes #1224

See that issue for a gif of the feature.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/OpenProjectEditorOnDefaultActionCommand.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Input;
+using Microsoft.VisualStudio.ProjectSystem.Input;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
+{
+    // Opens the Project file in the test editor when the user double-clicks or presses ENTER on the project file while its selected
+    [ProjectCommand(CommandGroup.UIHierarchyWindow, UIHierarchyWindowCommandId.DoubleClick, UIHierarchyWindowCommandId.EnterKey)]
+    [AppliesTo(ProjectCapability.AppDesigner)]
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/OpenProjectEditorOnDefaultActionCommand.cs]: This capability should be "OpenProjectFile".

Comment (davidwengier) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/OpenProjectEditorOnDefaultActionCommand.cs]: Thanks

**Comment by davkean** [comment]
@Pilchie @cartermp As discussed, our thoughts for this is that we'll turn this on for Preview 1 and monitor feedback on it.

**Comment by davkean** [comment]
tag @jviau - I'm guessing we'll want this in CPS long term.

**Pull request merged**